src/main/java/nextstep/subway/member/application/MemberService.java

```diff
@@ -33,4 +34,9 @@ public void updateMember(Long id, MemberRequest param) {
     public void deleteMember(Long id) {
         memberRepository.deleteById(id);
     }
+
+
+    public Member findMemberOrThrow(Long id) {
+        return memberRepository.findById(id).orElseThrow(IllegalArgumentException::new);
+    }
```

Comment:
@Transactional(readOnly = true) 를 사용해 보면 어떨까요? 😄 
```suggestion
    @Transactional(readOnly = true)
    public Member findMemberOrThrow(Long id) {
        return memberRepository.findById(id).orElseThrow(IllegalArgumentException::new);
    }
```

Reply:
find 부분에 대해서는 수정하였습니다.!